config/locales/en.yml

```diff
@@ -591,6 +591,14 @@ en:
     no_such_entry:
       heading: "No entry with the id: %{id}"
       body: "Sorry, there is no changeset with the id %{id}. Please check your spelling, or maybe the link you clicked is wrong."
+  context:
```

Comment:
If you moved this to `layouts.contextmenu` instead of here then you could just use `.directions_from` etc in the view to reference them.